.github/kokoro/steps/xilinx.sh

```diff
@@ -7,9 +7,99 @@
 #
 # SPDX-License-Identifier: ISC
 
-# Fix up things related to Xilinx tool chain.
+echo
+echo "========================================"
+echo "Xilinx proprietary toolchain setup."
+echo "----------------------------------------"
 
+echo
+echo "Fix up the Xilinx configuration directory"
+echo "----------------------------------------"
 ls -l ~/.Xilinx
 sudo chown -R $USER ~/.Xilinx
 
 export XILINX_LOCAL_USER_DATA=no
+echo "----------------------------------------"
+
+echo
+echo "Select Xilinx Vivado version"
+echo "----------------------------------------"
+(
+	set -e
+	cd /opt
+	if [ x"$XRAY_SETTINGS" = x"kintex7" ]; then
```

Comment:
We might add a new `env_vars` in the kokoro config to select which configuration uses which Vivado version?

Reply:
I guess we could in a future version.

Reply:
The same solution should hopefully work for custom GitHub Action runners...